docs/contributing.md

```diff
@@ -1,3 +1,5 @@
+<!-- The content of this file is manually kept in sync with ../CONTRIBUTING.md. There are formatting differences because this file uses Hugo templating. -->
```

Comment:
This could be mentioned:

> When you make a PR, a github action runs automatically and deploys a test website using your branch.

But it would be good to know also how to build & view the docs locally. For example adding this comment here in a "hugo template file", I'm not sure how it's going to look like when it's live before the gh action has finished deploying the changes that I pushed.

Reply:
You can clone the repo, use provided README file to set it up (granted, it's not the best manual, I am working on something better) and see it locally.

Reply:
Thanks!

I had to run `npm install` before `npm run dev` – so that's missing from https://github.com/python-poetry/website#local-development.

I get this in the terminal:
```
rollup v2.50.5
bundles src/app.js → assets/assets/app.js...
```

Where can I view the docs?

Reply:
It should be in the `public/` folder as a pile of files -- that being said, note that the action requires someone to add the `Documentation` tag, and presently that has to be a member of the Poetry team. It's really meant as a way for the team to view the rendered docs without a local clone, as opposed to a way to give feedback to the PR author.

We should focus on documenting how to build the website locally for this use case.

Reply:
I think I figured it out. Previously I was trying `poetry run python bin/website build --local ./poetry` which lead into some errors, and I ignored them – I shouldn't have :) Of course I didn't have `./poetry `, I have `../poetry`.

So, this worked:
```
cd website
poetry run python bin/website build --local ../poetry
hugo server -w
```

> Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)

And that answered my question about "Where can I view the docs?".

I can volunteer to document this in the contributing docs if it helps, let me know.

Reply:
checkout latest update to `package.json` and `npm scripts` provided in website. you can now just run `npm run dev` and both npm and hugo will run. There is also `Makefile` providing ready to use scripts for that. I don't see need for all this to be in Poetry contributing.md. Those are separate projects.

Reply:
Related: https://github.com/python-poetry/website/pull/48